ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,262 @@
+---
```

Comment:
```suggestion
---
eip: 7815
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
Thank you! Added.